packages/guides-restructured-text/src/RestructuredText/Directives/BaseDirective.php

```diff
@@ -92,6 +94,21 @@ public function processAction(
     ): void {
     }
 
+    /**
+     * Post processes a node created by the directive to apply common options
+     *
+     * @param DirectiveOption[] $options
+     */
+    final public function postProcessNode(Node $node, array $options): Node
```

Comment:
since this is moved to the rule, it would be wise to move this method to the rule, 

The directive class now has a method `getOption` that will simplify this method. You might want to have a look at that.